### PR TITLE
QTBSI-892: Update university list

### PR DESCRIPTION
**Pull request by lassesuom**
* Add new institutions to EDU backend vol. 36

### CODE REVIEW

File: world_universities_and_domains.json [hunk 1]
```diff
@@ -130977,5 +130980,149 @@
...
+    "domains": [
+      "vbithyd.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.ksriet.ac.in/"
+    ],
+    "name": "KSR Institute for engineering and technology",
```
Comment (Qt-nhannguyen) [inline world_universities_and_domains.json:131012]: I think we can capitalize some words to make them align with other university names
`KSR Institute for Engineering and Technology`

File: world_universities_and_domains.json [hunk 2]
```diff
@@ -130977,5 +130980,149 @@
...
+    "domains": [
+      "rkgit.edu.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://vbithyd.ac.in/"
+    ],
+    "name": "Vignana Bharathi Institute of science and Technology",
```
Comment (Qt-nhannguyen) [inline world_universities_and_domains.json:131000]: `Vignana Bharathi Institute of Science and Technology`

File: world_universities_and_domains.json [hunk 3]
```diff
@@ -130977,5 +130980,149 @@
...
+    "domains": [
+      "student.ztu.edu.ua"
+    ],
+    "country": "Ukraine"
+  },
+  {
+    "web_pages": [
+      "https://pstu.edu/en/"
+    ],
+    "name": "State Higher Educational Institution «Pryazovskyi State Technical University»",
```
Comment (Qt-nhannguyen) [inline world_universities_and_domains.json:131048]: I suggest the special chars `«` & `»` should be avoided for potential encoding/decoding issues.

`State Higher Educational Institution (Pryazovskyi State Technical University)`

File: world_universities_and_domains.json (hunk 1 above)
Comment (lassesuom) [reply world_universities_and_domains.json:131012]: Sounds good, I'll change that

**Review by Qt-nhannguyen — APPROVED** [review]
After all the correction, the PR can be merged 😄

**Pull request merged**